### DISCUSSION TITLE
Accuracy Of Data?

### DISCUSSION THREAD

**Discussion by AaronThomas22** [merchantprotocol/capital-audience-support#17] [Q&A]
@jonathonbyrdziak -> I just installed the pixel on my client's website. The client is a LOCAL home services contractor (roofing, siding, gutters, solar, windows). But, all the traffic that is listed in the identified visitor profiles are ALL out of state and not even near Pennsylvania. I could understand a few, but when the majority of the traffic to my client's site should be PA, MD, and DE, and absolutely no visitor profile is from any of those three or surrounding states has me worried. 

Is this a bug? Or is the data not accurate? Is it pulling from someone else's pixel? 

Attaching a screenshot and happy to do a Loom to to get this resolved.

<img width="893" alt="Screenshot 2023-01-10 at 6 21 56 AM" src="https://user-images.githubusercontent.com/105283193/211540532-6263d13e-c008-4c6b-8183-31b3aaa55164.png">

**Comment by jonathonbyrdziak** [comment]
Just released an announcement concerning this https://github.com/merchantprotocol/capital-audience-support/discussions/19
